src/app/clusters/service-area-server/service-area-server.cpp

```diff
@@ -0,0 +1,1101 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+#include "service-area-server.h"
+#include "service-area-delegate.h"
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Commands.h>
+#include <app/AttributeAccessInterfaceRegistry.h>
+#include <app/CommandHandler.h>
+#include <app/ConcreteCommandPath.h>
+#include <app/EventLogging.h>
+#include <app/InteractionModelEngine.h>
+#include <app/reporting/reporting.h>
+#include <app/util/attribute-storage.h>
+#include <app/util/util.h>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+void MatterServiceAreaPluginServerInitCallback(){};
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
```

Comment:
The code is already in chip::app::Clusters, why do we have to have these using statements?

Reply:
This has been fixed [here](https://github.com/project-chip/connectedhomeip/pull/33991/commits/d5d2d35f562cf729dec0382b3788cea0100e9660).